docs/en-us/workloadspread.md

```diff
@@ -3,79 +3,112 @@ title: WorkloadSpread
 called `subset`. Each domain should at least provide the capacity to run the replicas number of pods called `maxReplicas`.
-WorkloadSpread injects the domain config into the Pod by Webhook, and it also controls the order of scale in and scale out.
+WorkloadSpread injects the domain configuration into the Pod by Webhook, and it also controls the order of scale in and scale out.
 
 Currently, supported workload: `CloneSet`、`Deployment`、`ReplicaSet`.
 
-API definition：https://github.com/openkruise/kruise/blob/819125ddbd4fb4ffb5f3b1ecf03490349a8f6727/apis/apps/v1alpha1/workloadspread_types.go
-
-## WorkloadSpread Spec
+## WorkloadSpread Demo
 
 ```yaml
-// WorkloadSpreadSpec defines the desired state of WorkloadSpread.
-type WorkloadSpreadSpec struct {
-	// TargetReference is the target workload that WorkloadSpread want to control.
-	TargetReference *TargetReference `json:"targetRef"`
-
-	// Subsets describes the pods distribution details between each of subsets.
-	Subsets []WorkloadSpreadSubset `json:"subsets"`
-
-	// ScheduleStrategy indicates the strategy the WorkloadSpread used to preform the schedule between each of subsets.
-	// +optional
-	ScheduleStrategy WorkloadSpreadScheduleStrategy `json:"scheduleStrategy,omitempty"`
-}
+apiVersion: apps.kruise.io/v1alpha1
+kind: WorkloadSpread
+metadata:
+  name: workloadspread-demo
+spec:
+  targetRef:
+    apiVersion: apps/v1 | apps.kruise.io/v1alpha1
+    kind: Deployment | CloneSet
+    name: workload-xxx
+  subsets:
+    - name: subset-a
+      requiredNodeSelectorTerm:
+        matchExpressions:
+          - key: topology.kubernetes.io/zone
+            operator: In
+            values:
+              - zone-a
+    preferredNodeSelectorTerms:
+      - weight: 1
+        preference:
+        matchExpressions:
+          - key: another-node-label-key
+            operator: In
+            values:
+              - another-node-label-value
+      maxReplicas: 3
+      tolertions: []
+      patch:
+        metadata:
+          labels:
+            xxx-specific-label: xxx
+    - name: subset-b
+      requiredNodeSelectorTerm:
+        matchExpressions:
+          - key: topology.kubernetes.io/zone
+            operator: In
+            values:
+              - zone-b
+  scheduleStrategy:
+    type: Adaptive | Fixed
+    adaptive:
+      rescheduleCriticalSeconds: 30
 ```
-**Caution**：`targetRef`can not be mutated，and one workload can only correspond to one WorkloadSpread.
 
-### Subset
+`targetRef`: specify the target workload. Can not be mutated，and one workload can only correspond to one WorkloadSpread.
+
+## spec.subsets
+
+`subsets` consists of multiple domain called `subset`, and each topology has different configuration.
+
+### subset sub-fields
+
+- `name`: the name of `subset`, it is distinct in a WorkloadSpread, which represents a topology.
+
+- `maxReplicas`：the capacity of `subset`, and must be Integer and >= 0. There is no replicas limit while the `maxReplicas` is nil.
```

Comment:
the capacity of `subset` -> the replicas limit of subset
term 'capacity' can be misleading, it is not about the physical resource capacity but a manual specified limit or policy